provider_container_settings.json

```diff
@@ -11,7 +11,7 @@
         "name": "java",
         "binaryPath": "/jdtls/bin/jdtls",
         "initConfig": {
-            "location": "examples/java",
+            "location": "/analyzer-lsp/examples/java.jar",
```

Comment:
This is for testing, should be pulled out into something else until we can specify multiple initConfigs. Might be appropriate to file an issue to follow-up on this once that's possible.